### DISCUSSION TITLE
LTAR Question [Dec 28]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4754] [Discord Threads]
> 🙋‍♂️ **wobb**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### do u mean you want a row in `Recipes` links to multiple products, possibly the same product?    
####    
#### like the below?   
####    
#### Recipe A, [Product A]   
#### Recipe B, [Product A, Product B]   
#### Recipe C, [Product A, Product A, Product A]  
#  
> 🙋‍♂️ **olisinfo**  
#### we are on Digital ocean, how we can upgrade NocoDB  
#  
> 🙋‍♂️ **wobb**  
#### Hi thanks for replying, yes that's right.  
#  
> 🙋‍♂️ **wkw**  
#### Does a product only have one recipe ?  
#  
> 🙋‍♂️ **wobb**  
#### No, each product can belong to many recipes.  
#  
> 🙋‍♂️ **wkw**  
#### But why u need to choose the same product multiple times  
#  
> 🙋‍♂️ **wobb**  
#### e.g. A cake might need 2 x Egg but an omelette needs 4 x Egg  
#  
> 🙋‍♂️ **wkw**  
#### i got wt u mean. currently it s not supporting like this. it can only choose one  
#  
> 🙋‍♂️ **wkw**  
#### <@1046711021018038353> for upgrade, see https://docs.nocodb.com/getting-started/upgrading  
#  
> 🙋‍♂️ **wobb**  
#### Ah ok, thanks, will try something else.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1046565138595074048)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
